### DISCUSSION TITLE
trans-digital-cn/%E5%81%A5%E5%BA%B7%E6%8A%A4%E7%90%86/%E6%89%8B%E6%9C%AF/%E8%B5%B0%E8%BF%9B%E6%80%A7%E5%88%AB%E9%87%8D%E5%A1%91%E4%B8%AD%E5%BF%83%E9%87%8D%E6%96%B0%E8%AE%A4%E8%AF%86%E6%80%A7%E5%88%AB%E2%80%94%E2%80%94%E8%AE%BF%E4%B8%AD%E5%9B%BD%E5%8C%BB%E5%AD%A6%E7%A7%91%E5%AD%A6%E9%99%A2%E6%95%B4%E5%BD%A2%E5%A4%96%E7%A7%91%E5%8C%BB%E9%99%A2%E6%80%A7%E5%88%AB%E9%87%8D%E5%A1%91%E4%B8%AD%E5%BF%83%E9%A6%96%E5%B8%AD%E4%B8%93%E5%AE%B6%E9%99%88%E7%84%95%E7%84%B6%E5%8D%9A%E5%A3%AB_page/

### DISCUSSION THREAD

**Discussion by giscus** [project-polymorph/trans-digital-cn#2] [Announcements]
# trans-digital-cn/%E5%81%A5%E5%BA%B7%E6%8A%A4%E7%90%86/%E6%89%8B%E6%9C%AF/%E8%B5%B0%E8%BF%9B%E6%80%A7%E5%88%AB%E9%87%8D%E5%A1%91%E4%B8%AD%E5%BF%83%E9%87%8D%E6%96%B0%E8%AE%A4%E8%AF%86%E6%80%A7%E5%88%AB%E2%80%94%E2%80%94%E8%AE%BF%E4%B8%AD%E5%9B%BD%E5%8C%BB%E5%AD%A6%E7%A7%91%E5%AD%A6%E9%99%A2%E6%95%B4%E5%BD%A2%E5%A4%96%E7%A7%91%E5%8C%BB%E9%99%A2%E6%80%A7%E5%88%AB%E9%87%8D%E5%A1%91%E4%B8%AD%E5%BF%83%E9%A6%96%E5%B8%AD%E4%B8%93%E5%AE%B6%E9%99%88%E7%84%95%E7%84%B6%E5%8D%9A%E5%A3%AB_page/

None

https://transterminus.github.io/trans-digital-cn/%E5%81%A5%E5%BA%B7%E6%8A%A4%E7%90%86/%E6%89%8B%E6%9C%AF/%E8%B5%B0%E8%BF%9B%E6%80%A7%E5%88%AB%E9%87%8D%E5%A1%91%E4%B8%AD%E5%BF%83%E9%87%8D%E6%96%B0%E8%AE%A4%E8%AF%86%E6%80%A7%E5%88%AB%E2%80%94%E2%80%94%E8%AE%BF%E4%B8%AD%E5%9B%BD%E5%8C%BB%E5%AD%A6%E7%A7%91%E5%AD%A6%E9%99%A2%E6%95%B4%E5%BD%A2%E5%A4%96%E7%A7%91%E5%8C%BB%E9%99%A2%E6%80%A7%E5%88%AB%E9%87%8D%E5%A1%91%E4%B8%AD%E5%BF%83%E9%A6%96%E5%B8%AD%E4%B8%93%E5%AE%B6%E9%99%88%E7%84%95%E7%84%B6%E5%8D%9A%E5%A3%AB_page/

**Comment by yunwei37** [comment]
老早了x
